bin/clover/src/cloud-control-funcs/management/awsCloudControlDiscover.ts

```diff
@@ -1,12 +1,35 @@
+const getImportExceptions = (resourceId: string, region: string) => ({
```

Comment:
I don't believe you can put anything outside of the function itself; but you can put `function getImportExceptions(...)` inside the main() body itself--in fact, you can put it all the way at the bottom and still use it at the top!